scripted-tests/run/external-dependencies/src/main/scala/AppletTutorial.scala

```diff
@@ -0,0 +1,66 @@
+import java.awt.event.MouseListener
+import java.awt.event.MouseEvent
+import java.applet.Applet
+import java.awt.Graphics
+
+// We need linking errors, I assume we will never port java.applet._
```

Comment:
That's correct. Porting java GUI frameworks is out of scope for Scala Native. Lets change this comment to something less vague and more affirmative.